### PR TITLE
Feature/join room

### PR DESCRIPTION
**Pull request by TobiasMeissner2000**
#6

### CODE REVIEW

File: Estimator/Estimator/obj/Debug/net5.0/scopedcss/projectbundle/Estimator.bundle.scp.css [hunk 1]
```diff
@@ -8,6 +11,188 @@ body[b-l6xs2pb1i5] {
...
+    font-weight: bold;
+    padding: 20px 20px;
+    text-decoration: none;
+    text-shadow: 0px 5px 0px #ffffff;
+    width: 100px;
+    height: 100px;
+    text-align: center;
+}
+
+.extimateButton:hover[b-ry06bn30sb] {
```
Comment (LeoEcker) [inline Estimator/Estimator/obj/Debug/net5.0/scopedcss/projectbundle/Estimator.bundle.scp.css:184]: estimate

File: Estimator/Estimator/obj/Debug/net5.0/scopedcss/projectbundle/Estimator.bundle.scp.css [hunk 2]
```diff
@@ -8,6 +11,188 @@ body[b-l6xs2pb1i5] {
...
+    background: linear-gradient(to bottom, #f6f6f6 5%, #ffffff 100%);
+    background-color: #f6f6f6;
+}
+
+.extimateButton:active[b-ry06bn30sb] {
```
Comment (LeoEcker) [inline Estimator/Estimator/obj/Debug/net5.0/scopedcss/projectbundle/Estimator.bundle.scp.css:189]: estimate

File: Estimator/Estimator/obj/Debug/net5.0/scopedcss/projectbundle/Estimator.bundle.scp.css [hunk 3]
```diff
@@ -8,6 +11,188 @@ body[b-l6xs2pb1i5] {
...
+
+joinButton:hover[b-ry06bn30sb]:after {
+    width: 100%;
+}
+
+/*Button*/
+
+/* estimateButton */
+
+.extimateButton[b-ry06bn30sb] {
```
Comment (LeoEcker) [inline Estimator/Estimator/obj/Debug/net5.0/scopedcss/projectbundle/Estimator.bundle.scp.css:164]: estimate

File: Estimator/Estimator/obj/Debug/net5.0/scopedcss/bundle/Estimator.styles.css [hunk 4]
```diff
@@ -6,6 +9,188 @@ body[b-l6xs2pb1i5] {
...
+
+joinButton:hover[b-ry06bn30sb]:after {
+    width: 100%;
+}
+
+/*Button*/
+
+/* estimateButton */
+
+.extimateButton[b-ry06bn30sb] {
```
Comment (LeoEcker) [inline Estimator/Estimator/obj/Debug/net5.0/scopedcss/bundle/Estimator.styles.css:162]: estimate

File: Estimator/Estimator/Pages/JoinGroup.razor [hunk 5]
```diff
@@ -1,7 +1,76 @@
...
-<h3>JoinGroup</h3>
+<div>
+    <h3>JoinGroup</h3>
+
+    <div class="col" style="margin-top: 35px">
+        <div class="group, col">
+            <input type="text" required>
+            <span class="highlight"></span>
+            <span class="bar"></span>
+            <label>Your dispalyname</label>
```
Comment (LeoEcker) [inline Estimator/Estimator/Pages/JoinGroup.razor:11]: displayname

File: Estimator/Estimator/Pages/JoinGroup.razor [hunk 6]
```diff
@@ -1,7 +1,76 @@
...
+        </div>
+        <div class="group, col" style="margin-top: 35px">
+            <input type="text" required>
+            <span class="highlight"></span>
+            <span class="bar"></span>
+            <label>Room ID (case-sensitiv)</label>
+        </div>
+        <div class="col">
```
Comment (LeoEcker) [inline Estimator/Estimator/Pages/JoinGroup.razor:19]: why col?

File: Estimator/Estimator/Pages/JoinGroup.razor [hunk 7]
```diff
@@ -1,7 +1,76 @@
...
+        </div>
+    </div>
+
+
+</div>
+
+<div style="display: none">
+    <div class="row">
+        <div class="col">
+            <a href="#" class="extimateButton">0</a>
```
Comment (LeoEcker) [inline Estimator/Estimator/Pages/JoinGroup.razor:32]: estimate*

**Review by LeoEcker — APPROVED** [review]

File: Estimator/Estimator/Pages/JoinGroup.razor (hunk 6 above)
Comment (TobiasMeissner2000) [reply Estimator/Estimator/Pages/JoinGroup.razor:19]: colum

**Pull request merged**